smart_card_connector_app/src/window.css

```diff
@@ -30,6 +30,12 @@ button {
   font-weight: 400;
 }
 
+html,
+body {
+  width: 530px;
```

Comment:
BTW, does this affect the window's appearance in the App mode? Ideally we should avoid that.